### DISCUSSION TITLE
Glide Typing not working with debug releases

### DISCUSSION THREAD

**Discussion by brennoflavio** [florisboard/florisboard#1431] [Q&A] [answered]
FlorisBoard is an awesome keyboard, and I'm trying to learn more on how it works. For that, I'm trying to play with the debug builds that are available from the Github action. But I could not get glide typing working with that version for some reason that I don't understand. I activate the feature, and it simply does not work. F-droid version works as expected on my device.

Any advice on how to debug and get it working.

Again, thanks for the awesome work!

**Comment by patrickgold** [comment, marked as answer]
That's because since beta06 the keyboard UI uses a completely new internal keyboard logic and UI library and I wasn't able to re-implement everything again in that beta version. See the [changelog](https://github.com/florisboard/florisboard/releases/tag/v0.3.14-beta06) or the message on the Settings Home Screen for more info. Hope you can understand!

**Comment by brennoflavio** [comment]
Thanks for the answer, I missed that when reading the changelog.
